src/CreateMain.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import Main from "./Main";
+
+import styled from "styled-components";
+
```

Comment:
i see your importing styled components but not using it anywhere in this component. While it's not gonna break anything, you only need to import it into the components that have styled components.